### PR TITLE
Pre-release merge for docs 0.8.7

### PR DESCRIPTION
**Pull request by bmaltzan**

### CODE REVIEW

**Review by jweiskoff — APPROVED** [review]

**Review by afromme — APPROVED** [review]

**Review by jimentwood — APPROVED** [review]

**Review by rebamrich — APPROVED** [review]

**Review by mhl10 — APPROVED** [review]

**Review by helendwang — APPROVED** [review]

File: help/moderation/index.md [hunk 1]
```diff
@@ -85,23 +85,8 @@ arXiv may reclassify already announced papers if the moderators determine there
...
-
-
-The appeals process allows moderators additional time to reconsider a submission, but repeated appeals with no additional information cannot be considered. If you send an appeal and the moderators reach the same decision as they did initially, no further appeal should be requested. Extreme cases may be addressed to the appropriate advisory committee chair only: [physics](/help/physics/#AdvisoryCommittee);
-[mathematics](/help/math/#AdvisoryCommittee); 
-[computer science (CoRR)](/corr); 
-[quantitative biology](/help/q-bio/#AdvisoryCommittee); 
-[quantitative finance](/help/q-fin/#AdvisoryCommittee);
-[statistics](/help/statistics/#AdvisoryCommittee);
-[electrical engineering and systems science](/help/eess/#AdvisoryCommittee); or [economics](/help/econ/#AdvisoryCommittee).
+If you disagree with a moderation decision, you [may appeal](appeals). Please be sure to carefully [read through the appeals process](appeals) to ensure your request is fully considered.
```
Comment (SBBCornell) [inline help/moderation/index.md:88]:
```suggestion
If you disagree with a moderation decision, you may appeal. Please be sure to carefully [read through the appeals process](appeals) to ensure your request is fully considered.

```

Comment (SBBCornell) [reply help/moderation/index.md:88]: Per a11y criteria, having two links to the same place in close proximity can cause confusion for those using screen readers. I suggest the small change above so it's just one link to the appeals page.

Comment (SBBCornell) [inline help/moderation/index.md:88]:
```suggestion
If you disagree with a moderation decision, you may appeal. Please be sure to carefully [read through the appeals process](appeals) to ensure your request is fully considered.

```

**Review by SBBCornell — APPROVED** [review]
Looks great! I made one small suggestion (same suggestion on two pages) for accessibility compliance.

File: help/moderation/index.md (hunk 1 above)
Comment (jimentwood) [reply help/moderation/index.md]: looks good

**Pull request merged**